onebone/signal/filter.py

```diff
@@ -367,3 +367,118 @@ def bandstop_filter(
     b, a = butter(order, [low, high], btype="bandstop")
     signal = lfilter(b, a, signal, axis=axis)
     return signal
+
+
+"""hampel_filter.
+- Author: Sunjin Kim
+- Contact: sunjin.kim@onepredict.com
+"""
+
+
+def hampel_filter(x: np.ndarray, window_size: int, n_sigmas: float = 3) -> Tuple[np.ndarray, list]:
```

Comment:
docstring의 변수 이름과 다르네요. (다른 곳도 변경 필요)
`n_sigmas` -> `n_sigma`
```suggestion
def hampel_filter(x: np.ndarray, window_size: int, n_sigma: float = 3) -> Tuple[np.ndarray, list]:
```